hnn_core/parallel_backends.py

```diff
@@ -648,11 +649,7 @@ def __init__(self, n_procs=None, mpi_cmd='mpiexec'):
                          'mpi_child.py')
 
         # Split the command into shell arguments for passing to Popen
-        if 'win' in sys.platform:
-            use_posix = True
-        else:
-            use_posix = False
-        self.mpi_cmd = shlex.split(self.mpi_cmd, posix=use_posix)
+        self.mpi_cmd = shlex.split(self.mpi_cmd, posix=sys.platform != 'win32')
```

Comment:
what about `cygwin` as you mention in your comment above?

Reply:
I don't think people are going to be using Cygwin... but it seems to be able to use posix or win32 style-paths. So to write a fix for that platform, we'd need to be able to detect the style of path the user supplied and use correct parsing flag. It doesn't seem worth the effort unless we know of anyone who uses it...

Reply:
```suggestion
        use_posix = True if sys.platform != 'win32' else False
        self.mpi_cmd = shlex.split(self.mpi_cmd, posix=use_posix)
```

just a bit more explicit and clear

Reply:
Thanks!